backend/src/main/java/kg16/Utils.java

```diff
@@ -5,13 +5,22 @@
 public class Utils {
     private Utils() {}
 
-    private static final String MAC_REGEX = "^[0-9A-Fa-f]{2}(-[0-9A-Fa-f]{2}){5}$";
-    private static final Pattern MAC_PATTERN = Pattern.compile(MAC_REGEX);
+    private static final Pattern MAC_PATTERN = Pattern.compile("^[0-9A-Fa-f]{2}(-[0-9A-Fa-f]{2}){5}$");
+    private static final Pattern EMAIL_PATTERN = Pattern.compile("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$");
+    private static final Pattern PHONE_PATTERN = Pattern.compile("^\\+?[1-9]\\d{1,14}$");
 
     public static boolean isValidMacAddress(String mac) {
         return mac != null && MAC_PATTERN.matcher(mac).matches();
     }
 
+    public static boolean isValidEmail(String mac) {
+        return mac != null && EMAIL_PATTERN.matcher(mac).matches();
```

Comment:
I think "String mac" is some remnant from copy/pasting the isValidMacAddress? Maybe change to email/phone as parameter